reconstruction/cvt/src/main/java/org/jlab/rec/cvt/measurement/Measurements.java

```diff
@@ -297,7 +297,15 @@ private List<Surface> getClusterSurfaces(DetectorType type, List<Cluster> cluste
             if(cluster.getDetector()!=type) continue;
             int layer = MLayer.getType(type, cluster.getLayer()).getCVTLayer();
             Surface measure = cluster.measurement();
-            measure.hemisphere = Math.signum(cluster.center().y());
+            if(cluster.getDetector()==DetectorType.BMT) {
```

Comment:
isn't this necessary only for BMTC?

Reply:
it may be better to clone this method adding seed as an argument like it is done for the cosmic ray

Reply:
yes. And done...

Reply:
Changes done.